scrapers/coursebook.go

```diff
@@ -20,7 +20,7 @@ import (
 func ScrapeCoursebook(term string, startPrefix string, outDir string) {
 
 	// Load env vars
-	if err := godotenv.Load(); err != nil {
+	if err := godotenv.Load(".env.template"); err != nil {
```

Comment:
There is actually an error here I noticed after-the-fact (which I've fixed via [da5bab6](https://github.com/UTDNebula/api-tools/commit/da5bab6522491c7093f77ab2964c106d6c3c36b7)); this shouldn't be changed to read from `.env.template` since the template file is only used as a basis for creating an actual `.env` file (which is what this loads from by default)